librocksdb-sys/build.rs

```diff
@@ -309,6 +311,15 @@ fn try_to_find_and_link_lib(lib_name: &str) -> bool {
     false
 }
 
+fn stdlib_flag() -> String {
+    let lib = env::var("ROCKSDB_CXX_STDLIB").unwrap_or_else(|_| "-std=c++11".to_owned());
+    if !lib.starts_with("-std=") {
```

Comment:
What about to use a regex validation here ?

Reply:
what should it validate?

Reply:
Validate that value is something like `-std=c++14` or `-std=gnu++11`. If value is different than return `-std=c++11`. Because if user makes an error than he would get compile error. So reason is to prevent such cases.

Reply:
I'm thinking about this, and I wonder if it's not better to let the build fail if the flag is not valid. 

If you change this value, and you made a typo, and we use the default c++11, you'd be building with a std version that you don't want, which might be more unexpected, and you might not even notice.

If you change this value, and the build fails fast, you can identify the problem right away.